src/components/common/Extensions/Navigation/partials/NavItem/NavItem.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react';
+import { useIntl } from 'react-intl';
+import clsx from 'clsx';
+import styles from './NavItem.module.css';
+import NavLink from '~shared/Extensions/NavLink';
+import { NavItemProps } from '../../types';
+
+const displayName = 'common.Extensions.Navigation.NavItem';
+
+const NavItem: FC<NavItemProps> = ({ disabled = false, linkTo, label }) => {
+  const { formatMessage } = useIntl();
+
+  const labelText = typeof label === 'string' ? label : label && formatMessage(label);
+
+  return (
+    <NavLink aria-disabled={disabled} className={clsx(styles.navLink, '[&.active]:bg-gray-50')} to={linkTo}>
```

Comment:
Wouldn't it be better to move `styles.navLink` to NavLink component instead of passing them?

Reply:
No, because NavLink is like a logic component that will use in the header component also but with different styles.